src/AspNetCoreModuleV2/CommonLib/exceptions.h

```diff
@@ -29,17 +66,17 @@ inline __declspec(noinline) HRESULT CaughtExceptionHResult()
     }
     catch (std::system_error& exception)
     {
-        ReportException(exception);
+        ReportException(LOCATION_CALL exception);
         return exception.code().value();
     }
     catch (std::exception& exception)
     {
-        ReportException(exception);
+        ReportException(LOCATION_CALL exception);
         return HRESULT_FROM_WIN32(ERROR_UNHANDLED_EXCEPTION);
     }
     catch (...)
     {
-        ReportUntypedException();
+        ReportUntypedException(LOCATION_CALL_ONLY);
```

Comment:
I am not sure whether the APP and IIS worker process is still healthy and should continue process requests after this unhandled exception.